packages/core/src/view/style/Stylesheet.ts

```diff
@@ -178,60 +175,36 @@ export class Stylesheet {
    * @param name Name for the style to be stored.
    * @param style Key, value pairs that define the style.
    */
-  putCellStyle(
-    name: keyof CellStateStyles,
-    style: CellStateStyles[keyof CellStateStyles]
-  ) {
-    (this.styles[name] as any) = style;
+  putCellStyle(name: string, style: CellStateStyle) {
+    this.styles.set(name, style);
   }
 
   /**
-   * Returns the cell style for the specified stylename or the given
-   * defaultStyle if no style can be found for the given stylename.
+   * Returns the cell style for the specified baseStyleName or the given
+   * defaultStyle if no style can be found for the given baseStyleName.
    *
-   * @param name String of the form [(stylename|key=value);] that represents the style.
+   * @param cellStyle An object that represents the style.
    * @param defaultStyle Default style to be returned if no style can be found.
    */
-  getCellStyle(name: string, defaultStyle: CellStateStyles) {
-    let style = defaultStyle;
-
-    if (name.length > 0) {
-      const pairs = name.split(';');
-
-      if (style && name.charAt(0) !== ';') {
-        style = clone(style);
-      } else {
-        style = {} as CellStateStyles;
-      }
-
-      // Parses each key, value pair into the existing style
-      for (const tmp of pairs) {
-        const pos = tmp.indexOf('=');
+  getCellStyle(cellStyle: CellStyle, defaultStyle: CellStateStyle) {
```

Comment:
❓ can we add the return type in the method signature for consistency with other methods?

Reply:
I've been removing the return types in the method signature when IDEs didn't have problem figuring out the actual type.
But, as you said, `getCellStyle` didn't have a clear return type, so I added below for IDEs to correctly figure it out.
```js
let style: CellStateStyle;
```

Reply:
ok, thanks for the information.